### PR TITLE
[skip ci] Add delegates to .pullapprove.yml

### PR DESCRIPTION
**Pull request by gigawhitlocks**
Please comment if you are a component owner & I'll add delegates for you as requested.

### CODE REVIEW

**Comment by dougm** [comment]
@caglar10ur for vcsim/toolbox

File: .pullapprove.yml [hunk 1]
```diff
@@ -59,12 +60,21 @@ groups:
...
+      - caglar10ur
     conditions:
       files:
         include:
-          - "cmd/toolbox/*"
           - "cmd/vcsim/*"
-          - "pkg/vsphere/simulator/*"
+          - "pkg/vsphere/toolbox/*"
+
+  vcsim-toolbox:
```
Comment (dougm) [inline .pullapprove.yml]: just 'toolbox'.. this component isn't related to vcsim, its our version of in-guest vm tools

**Comment by caglar10ur** [comment]
Can you take care of https://github.com/vmware/vic/pull/3929#issuecomment-279822830 while at it?

File: .pullapprove.yml (hunk 1 above)
Comment (gigawhitlocks) [inline .pullapprove.yml]: oh my bad

**Comment by gigawhitlocks** [comment]
@caglar10ur re: #3929 Who should own `pkg/vsphere/vsan` ?

**Pull request merged**